src/app/inventory/store/stats.ts

```diff
@@ -56,7 +56,7 @@ const statsNoBar = [
 ];
 
 /** Stats that all armor should have. */
-const armorStats = [
+export const armorStats = [
```

Comment:
Am I missing anything or the "new" stats should be listed here?

Reply:
That change was in a different branch